### PR TITLE
improve documentation

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - document using composition to do additional configuration of S3 buckets
 - update recs for using `default_tags`
 - warn people on AWS security controls
 - `README` in `docs/` that indexes all the content
 - `README`s for each host platform

### Change implications

 - dependencies added/changed? **no**
 - something important to note in future release notes? **no**

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206231323063016
  - https://app.asana.com/0/0/1206231170020914
  - https://app.asana.com/0/0/1206162721159887
  - https://app.asana.com/0/0/1206221395701375

### CODE REVIEW

File: docs/development/alpha-features/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Alpha Features
+
+This directory documents some features that we consider "alpha" - available for use, but without any
```
Comment (P-A-R-U-S) [inline docs/development/alpha-features/README.md:3]: np: These ??

**Review by P-A-R-U-S — APPROVED** [review]
Looks good. One very minor comment.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Host Environments
+
+  - [AWS](aws/getting-started.md)
+  - [GCP](gcp/getting-started.md)
+
+
+### Deployment Environments
+Psoxy can be *hosted* in GCP or AWS, but it can be *deployed* to those host platforms from a variety
+of locations.
```
Comment (aperez-worklytics) [inline docs/README.md:15]:
```suggestion
of locations:
```

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  - local machine (macOS or a Linux such as Ubuntu likely to be most straightforward options)
+  - remote VM/container (eg, EC2 instance)
+  - CI/CD, such as GitHub Actions, Atlassian Bamboo, etc.
+  - [Terraform Cloud / Terraform Enterprise](terraform-cloud.md)
+  - Cloud Shell: [aws](aws/cloud-shell.md) (not recommended) / [gcp](gcp/cloud-shell.md)
```
Comment (aperez-worklytics) [inline docs/README.md:21]: just double checking: is aws cloud shell not recommended or both (aws and gcp) ? Also, I'd add a comment about why we don't recommend this approach here

File: docs/sources/hris/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# hris
+
+The Psoxy HRIS (human resource information system) connector is intended to sanitize data exported
+from an HRIS system which you intend to transfer to Worklytics. The expected format is a CSV file,
+as defined in the documentation for import data (see PDF docs on this from Worklytics, available at
```
Comment (aperez-worklytics) [inline docs/sources/hris/README.md:5]: missing link?

```suggestion
as defined in the documentation for import data (see PDF docs on this from Worklytics, available at [link]?
```

**Pull request merged**